extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/MultiValueScrutinizer.java

```diff
@@ -29,6 +51,7 @@ public void scrutinize(ItemUpdate update) {
         if (update.isNew()) {
             for (PropertyIdValue pid : propertyCount.keySet()) {
                 if (propertyCount.get(pid) == 1) {
+                    List<MultivalueConstraint> constraints = _fetcher.getConstraintsByType(pid, MULTI_VALUE_CONSTRAINT_QID).map(statement -> statement == null ? null : new MultivalueConstraint(statement)).collect(Collectors.toList());
```

Comment:
When does it make sense for `getConstraintsByType` to return a null value in its stream? Intuitively I think these should be filtered out upstream, no?